src/divforma.c

```diff
@@ -37,15 +37,15 @@ static jmp_buf jmp_error_ptr;
 // ����������������������������������������������������������������������������
 //      Estructuras
 // ����������������������������������������������������������������������������
-
+#if !defined(_WIN32) && !defined(WIN32)
```

Comment:
in src/divforma.c you have commented out code for tagRGBQUAD and tagBITMAPFILEHEADER. Do we need this or are you using another bitmap struct somewhere else in the msys version?